config/configmap/inferenceservice.yaml

```diff
@@ -150,7 +150,14 @@ data:
        },
        "s3": {
            "s3AccessKeyIDName": "AWS_ACCESS_KEY_ID",
-           "s3SecretAccessKeyName": "AWS_SECRET_ACCESS_KEY"
+           "s3SecretAccessKeyName": "AWS_SECRET_ACCESS_KEY",
+           "s3Endpoint": "s3.amazonaws.com",
+           "s3UseHttps": "1",
+           "s3Region": "",
```

Comment:
I was unsure about having empty values here since I couldn't see any other place in the configmap where this was done, but it makes sense to me to have them here so that users know all the possible configuration options without needing to look through the source code

Reply:
Make sense, maybe add a comment with the AWS/S3 reference you had in the other place.

Reply:
have added the quick links above the credentials entry in configmap